apps/omg/test/omg/state/core/metrics_test.exs

```diff
@@ -0,0 +1,47 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.State.Core.MetricsTest do
+  @moduledoc """
+  Tests functional behaviors of `OMG.State.Core.Metrics`.
+  """
+  use ExUnitFixtures
+  use ExUnit.Case, async: true
+
+  alias OMG.Eth.Encoding
+  alias OMG.State.Core
+  alias OMG.Utxo
+
+  require Utxo
+
+  @eth OMG.Eth.RootChain.eth_pseudo_address()
+  @not_eth <<1::size(160)>>
+  @tag fixtures: [:alice, :bob, :carol]
+
+  test "calculate metrics from state", %{alice: alice, bob: bob, carol: carol} do
+    utxos = %{
+      Utxo.position(2_835, 4076, 3) => %OMG.Utxo{amount: 700_000_000, currency: @eth, owner: alice},
```

Comment:
Can you pls make these utxo positions looks more real - someone could get wrong assumptions about block numbers and deposits. So non-deposit position blknum is divisible by 1_000 and deposit ones has 0, 0 in tx & o-index